Dockerfile

```diff
@@ -12,12 +12,13 @@ RUN apk --no-cache add \
     boost-dev \
     libxml2-dev \
     libuv-dev \
-    util-linux-dev
+    util-linux-dev \
+    perl
 
 WORKDIR /
 RUN git clone https://community.opengroup.org/osdu/platform/domain-data-mgmt-services/seismic/open-vds.git
 WORKDIR /open-vds
-RUN git checkout cbcd7b6163768118805dbcd080a5b0e386b82a6a
```

Comment:
Excellent that we finally can update to a proper release. 

I guess that commit "cbcd7b6163768118805dbcd080a5b0e386b82a6a" of open-vds was used for a specific reason. At least it was not trivial to update open-vds earlier. What is the reason and why is it okay now?